content/6_Advanced/MinCostFlow.problems.json

```
@@ -47,10 +47,9 @@
       "source": "CF",
       "difficulty": "Hard",
       "isStarred": false,
-      "tags": [],
+      "tags": ["Bitmasks", "MCF"],
```

```suggestion
      "tags": ["Bitmasks", "MinCostFlow"],
```
i think this is the name of the existing tag